### ISSUE TITLE
Highlight focused pane option breaks on UI zoom out

### ISSUE THREAD

**Issue by smingerson** [rstudio/rstudio#10215]
### System details

    RStudio Edition : Desktop 
    RStudio Version :  2021.09.0 Build 351
    OS Version      :  Windows 10
    R Version       :  4.1.1

### Steps to reproduce the problem

1. Open RStudio
1. Check "Highlight focused pane" under Global Options > Accessibility.
1. Shift focus between panes, noting the dashed blue outline around the focused pane.
1. Go to Global Settings > Appearance
2. Change the zoom to 80%. As 
3. Shift focus between panes. The focus outline is now gone on panes, and only appears on the focused icon within panes such as files or environment.
4. Reset zoom to 100% (or higher) o restore outline.

### Describe the problem in detail

Decreasing the UI zoom to from 100% to 80% (and presumably other values) in Appearance breaks the accessibility option "Highlight focused pane". The outline disappears. I tested this with a daily build from today (2021-12-09), and the issue persists.

### Describe the behavior you expected

The outline should be present for UI zooms lower than 100%. 

If anyone has a solution that can be achieved using CSS in a custom theme until this makes it into RStudio, I'd love to hear it.

-  [x] I have read the guide for [submitting good bug reports](https://github.com/rstudio/rstudio/wiki/Writing-Good-Bug-Reports).
- [x] I have installed the latest version of RStudio, and confirmed that the issue still persists.
- [ ] If I am reporting an RStudio crash, I have included a [diagnostics report](https://support.rstudio.com/hc/en-us/articles/200321257-Running-a-Diagnostics-Report).
- [x] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.

**Comment by ronblum** [comment]
@smingerson Thank you for raising this!

I can reproduce it using
- The same Windows environment
- RStudio Desktop 1.4.1717 and 2022.02.0+364

**Comment by gtritchie** [comment]
See also: https://github.com/rstudio/rstudio/issues/7888
